libraries/shortly/Makefile

```diff
@@ -10,3 +10,9 @@ test:
 	./rebar3 eunit
 	./rebar3 ct
 	echo "ALL TESTS PASSED"
```

Comment:
Mmm i don't know if I am a big fan of printing all test passed. I would remove this.

Reply:
Yeah this should never be done. Also this target will be removed in favor of test targets only in the root Makefile